cisc/commands.go

```diff
@@ -46,6 +46,12 @@ func init() {
 				Usage:   "check the health of the cothority",
 				Action:  idCheck,
 			},
+			{
+				Name:    "qrcode",
+				Aliases: []string{"qr"},
+				Usage:   "print out the qrcode of the identity",
```

Comment:
good idea - I'll add a README.md to the `identity`-directory.

The QRCode is used to connect a phone to the identity-skipchain. It contains an IP:Port and the skipchain-id where the identity is stored.